metadata/src/main/java/org/apache/kafka/controller/BrokerHeartbeatManager.java

```diff
@@ -107,6 +125,13 @@ int id() {
             return id;
         }
 
+        /**
+         * Returns the sorted directories which the broker currently has available
+         */
+        List<Uuid> directories() {
```

Comment:
Seems a bit awkward to have the field named differently than its accessor. Can we just call them both `directories` ?

Reply:
This was part of [KAFKA-15361](https://issues.apache.org/jira/browse/KAFKA-15361) (https://github.com/apache/kafka/pull/14902) and is now gone since we removed dirs from the heartbeat manager.